server/frontend/src/components/Covmanager/Summary.vue

```diff
@@ -0,0 +1,539 @@
+<template>
+  <div id="main" class="panel panel-default">
+    <div class="panel-heading">
+      <i class="bi bi-file-bar-graph"></i> Report Summary -
+      <span>{{ chart.data.created }}</span>
+    </div>
+    <template v-if="loading">
+      <div class="loader"></div>
+      <template v-if="loading_incomplete">
+        <div class="loader-msg">
+          <span
+            >The requested summary is being generated by the server, please
+            wait...</span
+          >
+        </div>
+      </template>
+    </template>
+    <template v-else>
+      <div class="panel-body">
+        <button class="btn btn-default" @click="refresh()">
+          Refresh this Report Summary
+        </button>
+        <button class="btn btn-default" @click="textual()">
+          View Textual Report
+        </button>
+      </div>
+      <div class="panel-body">
+        <div id="graph-container" style="margin: 0 auto; width: 1280px">
+          <div style="padding-bottom: 20px">
+            <span>Maximum Display Depth: </span>
+            <vue-slider
+              v-model="chart.level"
+              :max="chart.maxlevel"
```

Comment:
This is not showing available levels, only current level?
eg. old:
![image](https://github.com/user-attachments/assets/242f6707-4d80-4ff1-a30e-86f7b39697f2)

new:
![image](https://github.com/user-attachments/assets/f04e40bd-5a20-477b-8cc7-20354a41d54f)

Reply:
You’re correct, this is currently showing only the current level. We’ll update it to display all available levels as well. Thank you for pointing that out!